docs/java/extensions.md

```diff
@@ -9,20 +9,15 @@ MetaDescription: Popular Java extensions for Visual Studio Code
 ---
 # Java extensions for Visual Studio Code
 
-Visual Studio Code provides many features as a Java language editor. You can also install VS Code extensions to add more features for Java development.
+Thanks to the great Java community around Visual Studio Code, you can use a wide range of extensions to enhance your Java development experience
 
 > **Tip:** To learn how to install and manage your extensions, refer to the [extension documentation](/docs/editor/extension-marketplace.md).
 
-Thanks to the great Java community around VS Code, you can search for a large number of Java extensions easily within VS Code:
+On this page, we will recommend a list of popular extensions that might help you for different scenarios.
 
-1. Go to the **Extensions** view (`kb(workbench.view.extensions)`).
-2. Filter the extensions list by typing "java".
-
-![Java Extensions](images/extensions/extensions.png)
+## Fundamental Java Development
 
-## Recommended extensions
-
-To help set up Java on VS Code, there is the Microsoft [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack), which contains these popular extensions:
+If you are looking for core Java development experience on Visual Studio Code (including Java code auto-completion, running / debugging / testing Java application, Java project management, etc), we recommend the [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack). This extension pack contains a list of popular Java extensions for fundamental Java development:
```

Comment:
Maybe add a `Install Extension Pack for Java` button like what we have in other docs?

Reply:
it's on line 29